### ISSUE TITLE
"model wasn't compiled" during fine-tuning

### ISSUE THREAD

**Issue by Ruslanmlnkv** [pyannote/pyannote-audio#54]
Hello! I have my own dataset with ~1500 speakers (3-10 minutes for each). I tried to train models with pyannote and have some results now.
1. Models works well on audio from the same devises (train and test stages), but the system quality falls very low on audio from different devices. I understand that it's may be actual only for my database. Do you test your model the same way or do you observe the same effects? 
2. The quality grew when I have used 20 MFCC instead 11.
3. Pre-training model works on Russian language, EER around 20% (60 speakers). 
4. I tried to fine-tune pre-training model. I used "--start" option for that, but got an error "model wasn't compiled". It fixed by adding:
`embedding.compile(optimizer=optimizer, loss=precomputed_gradient_loss)`
in 306 line base_autograd.py file, but I'm not sure that it's correct.

Thanks!

**Comment by hbredin** [comment]
Not sure Github is the best place to discuss experimental results: would you mind sending me an email at bredin__at__limsi.fr with more details on your experiments?

Concerning (4.), I will try to have a look at this (I will rename this issue accordingly). Was this an "error" or a "warning"?

**Comment by Ruslanmlnkv** [comment]
It is an error.
![2017-09-22 13 04 04](https://user-images.githubusercontent.com/22793531/30739597-a65b1cf4-9f96-11e7-8c69-74e56c3ace5d.png)

**Comment by hbredin** [comment]
`develop` branch dropped Keras completely in favor of `pytorch` and in preparation of version 1.0 release (around or before Interspeech 2018). 

Closing.
